### PR TITLE
yaft: init at 0.2.9

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/yaft/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  };
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/man}
+    DESTDIR=$out PREFIX=$out/bin/ MANPREFIX=$out/share/man/
+  '';
+
+  meta = {
+    homepage = "https://github.com/uobikiemukot/yaft";
+    description = "yet another framebuffer terminal";
```
Comment (rycee) [inline pkgs/applications/misc/yaft/default.nix]: The "y" in "yet" should be [capitalized](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes).

**Comment by rycee** [comment]
Maybe you forgot to add an attribute in `all-packages.nix`?

**Comment by rycee** [comment]
Hmm, this didn't build very well for me. Well it built but never installed anything but a few empty directories. I changed the expression a little:
```diff
diff --git a/pkgs/applications/misc/yaft/default.nix b/pkgs/applications/misc/yaft/default.nix
index a1708c6..9d717e0 100644
--- a/pkgs/applications/misc/yaft/default.nix
+++ b/pkgs/applications/misc/yaft/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, pkgconfig, ncurses, readline }:
+{ stdenv, fetchFromGitHub, ncurses }:
 
 stdenv.mkDerivation rec {
   version = "0.2.9";
@@ -11,10 +11,9 @@ stdenv.mkDerivation rec {
     sha256 = "0l1ig8wm545kpn4l7186rymny83jkahnjim290wsl7hsszfq1ckd";
   };
 
-  installPhase = ''
-    mkdir -p $out/{bin,share/man}
-    DESTDIR=$out PREFIX=$out/bin/ MANPREFIX=$out/share/man/
-  '';
+  buildInputs = [ ncurses ];
+
+  installFlags = [ "PREFIX=$(out)" "MANPREFIX=$(out)/share/man/" ];
 
   meta = {
     homepage = "https://github.com/uobikiemukot/yaft";
```
and with that it appears to build and install OK.

**Comment by matthiasbeyer** [comment]
Would be way easier if you would have pushed a commit that I can cherry-pick. But well, fixed your suggestions.

**Comment by rasendubi** [comment]
@matthiasbeyer, you doesn't seem to apply the patch. I have opened a PR with the fixes: https://github.com/matthiasbeyer/nixpkgs/pull/5.

**Comment by matthiasbeyer** [comment]
@rasendubi 10 hrs ago I was in bed sleeping, and I have a life besides my computer. But here you go!

**Comment by rycee** [comment]
Squashed and rebased into master in commit 904bcb391dc41717fedeefc07c63b5f8c4e6b88a.

**Pull request closed without merging**